Modules/mmapmodule.c

```diff
@@ -1282,7 +1282,7 @@ new_mmap_object(PyTypeObject *type, PyObject *args, PyObject *kwdict)
                                 "tagname",
                                 "access", "offset", NULL };
 
-    if (!PyArg_ParseTupleAndKeywords(args, kwdict, "in|ziL", keywords,
+    if (!PyArg_ParseTupleAndKeywords(args, kwdict, "in|OiL", keywords,
```

Comment:
Use "U" instead of "O". It does the type check for you.

Reply:
No, "U" should not be used here. `None` must be accepted as an argument for `tagname` (as [documented](https://docs.python.org/3.9/library/mmap.html#mmap.mmap)).

Reply:
Okay, right you are. 

Using `tagname=""` should be equivalent. Maybe map the empty string case to `m_obj->tagname = NULL`, as the current code does. But, in practice passing an empty string is effectively the same. The kernel only assigns an object name if it has non-zero length.